Frameworks/Foundation/NSOrderedSet.mm

```diff
@@ -16,44 +16,41 @@
 
 #include "Starboard.h"
 #import <StubReturn.h>
+#include <vector>
 
-#include "Foundation/NSOrderedSet.h"
+#import <Foundation/NSOrderedSet.h>
+#import <Foundation/NSException.h>
+
+@implementation NSOrderedSet {
+    std::vector<StrongId<id>> setContainer;
```

Comment:
Yes. I believe that we should be using both an array and a map here. The array is used for ordered operations while the map is used for set operations.

---
In reply to: [61937416](https://github.com/Microsoft/WinObjC/pull/472#discussion_r61937416) [](ancestors = 61937416,61937213)